tracer/src/Datadog.Trace/Debugger/Instrumentation/AsyncMethodDebuggerInvoker.cs

```diff
@@ -151,7 +159,9 @@ private static void BeginEndMethodLogArgs(ref AsyncMethodDebuggerState asyncStat
             for (var index = 0; index < kickOffMethodArguments.Length; index++)
             {
                 ref var argument = ref kickOffMethodArguments[index];
-                if (argument == default)
+                if (argument == default ||
```

Comment:
See previous note about extracting this predicate into a method

Reply:
Fixed in d93f31f39ab9606ae745e1c8547fc5e7e0a6b757